README.md

```diff
@@ -42,6 +42,17 @@ TRELLO_AUTO_ASSIGN - If set to '1', assigns sender with created card
 TRELLO_RESPONSE_PREFIX - Specify response message (default is 'Created')
 ```
 
+## Valid Value
+### dd \<due_date\>
+`YYYY-MM-DD` or `YYYY-MM-DD HH:MM`
+
+Example
+
+```
+dd 2016-01-01
+dd 2016-01-01 01:02
+```
```

Comment:
Can you add a complete instruction to the **Usage** section?

e.g.

    ## Usage
    
    ```
    @ruboty trello b <board_name> l <list_name> (lb <label_name>) (dd <due_date>) c <card_name>
    ```

     e.g.

    ```
    @ruboty trello b development l icebox c something to do
    @ruboty trello b development l icebox dd 2016-01-01 01:02 c something to do
    ```